sklearn/random_projection.py

```diff
@@ -436,15 +436,15 @@ class GaussianRandomProjection(BaseRandomProjection):
         very conservative estimated of the required number of components
         as it makes no assumption on the structure of the dataset.
 
-    eps : strictly positive float, optional (default=0.1)
+    eps : strictly positive float, default=0.1
```

Comment:
```suggestion
    eps : float, default=0.1
```
The information regarding the value of the `eps` can go in the description below